docs/source/contribute.rst

```diff
@@ -0,0 +1,54 @@
+.. _contribute:
+
+Contributing to Cartopy
+=======================
+
+Cartopy is an open-source project that welcomes contributions from anyone. A good place
+to start contributing is by fixing typos, adding examples, or improving the explanations
+of the functions.
```

Comment:
Until #1919 is resolved, the CLA should probably be mentioned up front.

Reply:
Good point, but I'm not sure how much detail we really want to go into here as I find the more text we start including the less likely people are to contribute! What about this as a compromise on more info, but not scaring people away?

"More information about the copyright and license agreement can be found on the `SciTools Governance page <https://scitools.org.uk/organisation.html>`_."